packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -57,34 +52,34 @@ export class FileTreeWidget extends TreeWidget {
         return classNames;
     }
 
-    protected renderIcon(node: TreeNode, props: NodeProps): h.Child {
+    protected renderIcon(node: TreeNode, props: NodeProps): React.ReactNode {
         if (FileStatNode.is(node)) {
-            return h.div({
-                className: (node.icon || '') + ' file-icon'
-            });
+            return <div className={(node.icon || '') + ' file-icon'}></div>;
         }
         // tslint:disable-next-line:no-null-keyword
         return null;
     }
 
-    protected onAfterAttach(msg: Message): void {
-        super.onAfterAttach(msg);
-        this.addEventListener(this.node, 'dragenter', event => this.handleDragEnterEvent(this.model.root, event));
```

Comment:
Should not we handle `dragenter` event as well, as before?

Reply:
Did not work before because use of native events. Should work now again.